protocols/radius/src/main/java/org/opennms/protocols/radius/monitor/RadiusAuthMonitor.java

```diff
@@ -29,6 +29,8 @@
 package org.opennms.protocols.radius.monitor;
 
 import java.net.InetAddress;
+import java.security.Provider;
+import java.security.Security;
```

Comment:
it's a small thing, but are these imports needed anymore now that they're in the static `RadiusUtils` call instead?
Otherwise this looks much cleaner, thanks!